wdl/tasks/Longbow.wdl

```diff
@@ -4,7 +4,7 @@ import "Structs.wdl"
 import "Utils.wdl"
 import "MASSeq.wdl" as MAS
 
-struct LongbowModelUmiAdjustmentParams {
+struct LongbowModelParams {
```

Comment:
Just a strange thought: why not make this a json file, and have a parser?
Don't handle this in this PR.

Reply:
Yeah - I want to remove it altogether and use calls to `Longbow` itself.  That way we're not defining things in more than one place.